testbed/tests/log_test.go

```diff
@@ -24,7 +24,6 @@ import (
 )
 
 func TestLog10kDPS(t *testing.T) {
-	flw := testbed.NewFluentBitFileLogWriter(testbed.DefaultHost, testbed.GetAvailablePort(t))
```

Comment:
We need to also remove `testbed.NewFluentBitFileLogWriter` and copy it to contrib correct?

Reply:
> We need to also remove `testbed.NewFluentBitFileLogWriter` and copy it to contrib correct?

See https://github.com/open-telemetry/opentelemetry-collector/pull/2795#discussion_r600934068 and https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/2729/commits/1365a25b93346fd6cd677ca921e222745d939c97

Reply:
I am confused the code for `NewFluentBitFileLogWriter` is still in this repo:
https://github.com/open-telemetry/opentelemetry-collector/blob/main/testbed/testbed/senders.go#L628

And it is not removed.

Reply:
Ah, my apologies - I misread. Yes, `testbed.NewFluentBitFileLogWriter` will still need to be removed and copied to contrib - since I've already started moving the tests over in my PRs, I'll handle it there - I'll try to get those changes committed tomorrow AM and update here since it overlaps.